lib/config/massage.spec.ts

```diff
@@ -26,6 +26,24 @@ describe('config/massage', () => {
       });
     });
 
+    it('massages enabledManagers', () => {
+      const config: RenovateConfig = {
+        enabledManagers: ['npm', 'custom', 'custom.regex'],
+      };
+      expect(massage.massageConfig(config)).toEqual({
+        enabledManagers: ['npm', 'custom'],
+      });
+    });
+
+    it('massages matchManagers', () => {
+      const config: RenovateConfig = {
+        matchManagers: ['npm', 'custom', 'custom.regex'],
+      };
+      expect(massage.massageConfig(config)).toEqual({
+        matchManagers: ['npm', 'custom'],
```

Comment:
```suggestion
        matchManagers: ['npm', 'custom.regex'],
```
I think `custom` shouldn't be here based on previous discussions 🤔

Reply:
Maybe keep it there till another PR comes along which can handle regex?

Reply:
I think we should migrate 1:1 to `custom.regex` as we have currently only one custom manager. 

@rarkins @viceice  WDYT?

Reply:
The migration is the same as you want:
`regex` -> `custom.regex`

Reply:
> I thought of keeping both till regex matching is implemented.

IMO no necessary as we have only one currently and with `jsonata` a second one in the near future.